backend/src/main/kotlin/blue/mild/covid/vaxx/routes/PatientRoutes.kt

```diff
@@ -126,7 +126,7 @@ fun NormalOpenAPIRoute.patientRoutes() {
     }
 
     // admin routes for registered users only
-    authorizeRoute(requireOneOf = setOf(UserRole.ADMIN, UserRole.DOCTOR)) {
+    authorizeRoute(requireOneOf = setOf(UserRole.ADMIN, UserRole.DOCTOR, UserRole.RECEPTIONIST)) {
```

Comment:
RECEPTIONIST má tímhle práva i na put a delete, to nevím jestli chceme

Reply:
Jo to je ok. Ty admin pracovnici maji pacienty editovat a kdyby na to meli na FE tlacitko, tak by je mohli i mazat. Naopak, jeste jsem pridal recepcni aby mohla potvrdit spravnost infromaci.